package.json

```diff
@@ -41,7 +41,7 @@
     "federalist": "npx gulp build",
     "lint": "gulp lintJS lintSass && npm run prettier:scss",
     "prestart": "npx gulp build",
-    "proof": "bundle exec htmlproofer --assume-extension --disable-external --allow-hash-href --alt-ignore '/.*/' --file-ignore '/developers/' ./_site",
+    "proof": "bundle exec htmlproofer --allow-hash-href=true --enforce-https=false --allow-missing-href=true --ignore-empty-alt=true --ignore-files '/_site/whats-new/updates/2017/'  --ignore-status-codes 403,429 ./_site",
```

Comment:
Reference: https://github.com/gjtorikian/html-proofer/blob/main/bin/htmlproofer

- `--ignore-files '/_site/whats-new/updates/2017/'` was included in this way because of some outdated references in news and updates from 2017. It seemed like a can of worms to try to remediate these, especially when the linked sites no longer exist, so I chose to ignore.
- `--ignore-status-codes 403,429` These status errors popped up only when attempted to access via htmlproofer. I was able to confirm that these links did work when accessed manually.

Reply:
Only 403's on `help.github.com` links (there are two). Not sure why, they work when testing manually.

Only question on this flag `--allow-missing-href=true`, do we get a lot of errors with this disabled?

Reply:
Last check there are 90+ errors from empty href tags, all appearing to originate from the USWDS build. (Specifically, they are originating from summary box, identifier, and docs template.) It looks like we would need to update the component twig in USWDS and add some defaults. Want me to open an issue to repair those with a note we should circle back and remove this flag once fixed? Is this a blocker?

Reply:
Yes, let's create an issue to track in USWDS. Thanks for that info!

Reply:
Issue created to track empty hrefs in USWDS: https://github.com/uswds/uswds/issues/4907

Reply:
Thanks for creating that issue, resolving this thread.